deprecation-notice.php

```diff
@@ -0,0 +1,47 @@
+<?php
+// Add an admin notice for outdated plugin version
+add_action('admin_notices', 'notify_plugin_update');
+function notify_plugin_update() {
+    $dismissed_timestamp = get_option('onesignal_deprecation_notice_dismissed_time');
+    $time_passed = !$dismissed_timestamp || (time() - $dismissed_timestamp) > (7 * DAY_IN_SECONDS);
```

Comment:
Do you think 7 days is regular enough for the warning to be shown again, after being dismissed?

Reply:
Yeah, I think seven days should be enough. We don't really have a way to know whether they've actually migrated so we don't want to be too bothersome with it